tests/integration/src/transfer_tests.rs

```diff
@@ -277,3 +287,301 @@ fn make_transfer_successful() {
 
     assert!(all_have_transaction_hash);
 }
+
+#[test]
+fn make_icrc1_transfer() {
+    let TestEnv {
+        mut env,
+        canister_ids,
+        // controller,
+        ..
+    } = setup_new_env();
+
+    let beneficiary_id = user_test_id(1);
+
+    // register user
+    let res: (ApiResult<MeResponse>,) =
+        update_candid_as(&env, canister_ids.station, WALLET_ADMIN_USER, "me", ()).unwrap();
+    let user_dto = res.0.unwrap().me;
+
+    let ledger_controller = Principal::from_slice(&[99; 29]);
+
+    let token_ledger_canister_id = deploy_icrc1_token(
```

Comment:
nit: this could pottentially be moved to a setup util method

Reply:
I'd wait and see how often we need to have an icrc1 token deployed, my assumtion is not often.